src/zenml/integrations/whylogs/steps/__init__.py

```diff
@@ -0,0 +1,19 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from zenml.integrations.whylogs.steps.whylogs_profiler import (
```

Comment:
Not sure if these are necessary either. Are you perhaps doing it to make the namespace easy to navigate?

Reply:
I'm merely applying what I thought to be a convention used in many other integrations, one that I particularly like too, because instead of:

```from zenml.integrations.whylogs.steps.whylogs_profiler import WhylogsProfilerStep```

the end user can use the shorter form:

```from zenml.integrations.whylogs.steps import WhylogsProfilerStep```